tests/foreman/cli/test_satellitesync.py

```diff
@@ -408,6 +408,23 @@ def tearDown(self):
         super(ContentViewSync, self).tearDown()
         ssh.command('rm -rf {}'.format(self.export_dir))
 
+    def assert_exported_cvv_exists(self, content_view_name, content_view_version):
+        """Verfiy an exported tar exists
+
+        :return: The path to the tar (if it exists).
+        """
+        exported_tar = '{0}/export-{1}-{2}.tar'.format(
+            self.export_dir, content_view_name, content_view_version)
+        result = ssh.command("[ -f {0} ]".format(exported_tar))
+        self.assertEqual(result.return_code, 0)
+        return exported_tar
+
+    def assert_import_export_packages_match(self, importing_cvv_id, exporting_cvv_id):
```

Comment:
I don't know where are you consuming this function?

Reply:
Hmm, I think I added this with the intent of using it in future tests, I didn't end up using it in this test, but probably will in the future. Can we keep it?